gcache/src/GCache_seqno.cpp

```diff
@@ -114,8 +114,15 @@ namespace gcache
 
         bool   loop(false);
 
+        bool sleep_a_bit = false;
```

Comment:
Let's keep the declaration/initialization style consistent in the file.

Reply:
Fixed.